recipe/guidata-version.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
+index b690d0c..d4ef6fe 100644
+--- a/setup.py
++++ b/setup.py
+@@ -247,7 +247,7 @@ setup(
+     install_requires=[
+         "NumPy>=1.3",
+         "SciPy>=0.7",
+-        "guidata==3.1",
++        "guidata>=3.5.2,<=3.9.0",

```

Comment:
This is also better for us to track and understand changes. The pip-check removal alone will be lost to git history, this patch will be here until we can to remove it.